cfme/tests/intelligence/chargeback/test_resource_allocation.py

```diff
@@ -34,7 +36,7 @@
     pytest.mark.meta(blockers=[BZ(1511099, forced_streams=['5.9', '5.8'],
                                   unblock=lambda provider: not provider.one_of(GCEProvider)),
                                ]),
-    pytest.mark.provider([SCVMMProvider], scope='module',
+    pytest.mark.provider([InfraProvider], scope='module', selector=ONE_PER_TYPE,
```

Comment:
Question: Please explain this replace. can we have both of them in these tests

Reply:
I added support for SCVMM through another PR.
I'm adding support for the other providers through this PR.
I have also provided explanation in the module itself.